### DISCUSSION TITLE
Longhorn formatted volume after couple of power failures

### DISCUSSION THREAD

**Discussion by bazuchan** [longhorn/longhorn#5731] [General]
I had a couple of power failures on part of my cluster and it resulted in two of my volumes got formatted.

`I0321 12:48:34.173952   16642 mount_linux.go:449] Disk "/dev/longhorn/pvc-438fa7d0-abb1-4f83-a503-0e92dcec1247" appears to be unformatted, attempting to format as type: "ext4" with options: [-F -m0 /dev/longhorn/pvc-438fa7d0-abb1-4f83-a503-0e92dcec1247]`

One of this volumes was mounted read-only all the time and no way it got corrupted, I think it was iscsi issue. Question is how to disable this behavior? So longhorn will never format a volume after initial formatting when it was created?

**Comment by derekbit** [comment]
RWO or RWX volume?
Can you provide the support bundle?

**Comment by bazuchan** [comment]
ReadWriteOnce
It happened a month ago, will support bundle be still useful?
And my question is how to disable this behavior, not to get what caused it.

**Reply by derekbit** [reply]
OK. I think the information are all gone. What's your LH version?

**Comment by bazuchan** [comment]
v1.4.0

**Reply by derekbit** [reply]
Can you copy one of the volume's replicas and check if it is mountable by the copied one as the steps in https://longhorn.io/docs/1.4.0/advanced-resources/data-recovery/export-from-replica/?

**Reply by bazuchan** [reply]
Yes, it is mountable and data is there.
